### DISCUSSION TITLE
关于深渊查询输出异常的问题

### DISCUSSION THREAD

**Discussion by Pyram1dCat** [Arondight/Adachi-BOT#460] [Q&A] [answered]
我是群主，在config.yml文件中添加了我和其中一个群员A的cookies，cookies包括：
`_MHYUUID; mi18nLang; _ga_831VBKXN1V; _ga_M6C431156S; _ga_YQPW66MJ73; _ga_4PPV2TWM03; _ga_HKTGWLY8PN; _ga=GA1.2.1500721659.1631574165; _gid; ltoken; ltuid=93350610; cookie_token; account_id`
添加完cookies之后在终端用npm run init初始化，终端输出运行状态：
![image](https://user-images.githubusercontent.com/95394240/144351710-da6c535f-796c-492b-8add-6278d2baef23.png)

之前我和群员A都已经绑定了米游社通行证ID，并且本期深渊(2021.12.1~2021.12.16)分别打到35星和36星。在群内发送指令查询深渊：
这是我查询的结果：
![image](https://user-images.githubusercontent.com/95394240/144351096-63e90ed1-6e4d-4157-8d8a-89083d1f687b.png)
输出正常。
这是群员A查询的结果：
![image](https://user-images.githubusercontent.com/95394240/144351152-d6f1491a-ce21-46c3-9ba0-f099d4697558.png)
输出没有下面每一层的角色使用。
再让群员A自己输入指令查询深渊，结果也是这样。在群员A的米游社设置中已经打开允许查看角色等（那两个滑块）。另外，机器人输出我的box是正常的，输出群员A的box会缺少很多角色，就像下图这样：
![image](https://user-images.githubusercontent.com/95394240/144352168-6706244f-26db-4987-85e2-705ebf89ff41.png)

请问是为什么会出现这种异常啊。直觉上感觉是很低级的问题，但我暂时找不出解决方法。

**Comment by Arondight** [comment]
更新代码

**Reply by Pyram1dCat** [reply]
更新后问题已解决，多谢！

**Comment by mark9804** [comment, marked as answer]
深渊和米游社API返回数据有变动，请参考 #436
